### ISSUE TITLE
Manual Schedule is not configured correctly.

### ISSUE THREAD

**Issue by cgardens** [airbytehq/airbyte#258]
## Expected Behavior
* For a manual connection, the UI should send a request with the schedule as null.

## Current Behavior
* For a manual connection, the UI is sending a scheduler with 0 minutes (which means the sync is scheduled all the time).
```
dataline-server | {"sourceImplementationId":"8dd8acc7-5e96-4a49-b400-83f6dcc7c976","destinationImplementationId":"956f182a-1235-4d6c-8ee7-8debb66ed74e","syncMode":"full_refresh","schedule":{"units":0,"timeUnit":"minutes"},"status":"active"}
```

## Steps to Reproduce
1. Create a connection with frequency set to manual

**Comment by Jamakase** [comment]
This is already addressed in PR #257

**Comment by cgardens** [comment]
fixed.
